rest/route/host_create_test.go

```diff
@@ -320,7 +307,6 @@ func TestHostCreateHandler(t *testing.T) {
 
 	c := apimodels.CreateHost{
 		Distro:              "archlinux-test",
-		CloudProvider:       "ec2",
```

Comment:
I think this should be fine but can you verify with evergreen all-configs that we're not going to start getting validation errors related to cloud provider being missing? [DEVPROD-14743](https://jira.mongodb.org/browse/DEVPROD-14743) makes me worried we're going to error instead of warn, as we're supposed to.

Reply:
I double checked with this binary and with the current, but the `provider` parameter is not required for the command. The only validation we had before (this change removes it), is it must have been `ec2` or `docker`, which now it will just ignore the parameter